pom.xml

```diff
@@ -76,7 +76,12 @@
         <xmlsec.version>2.0.8</xmlsec.version>
         <jts.version>1.12</jts.version>
         <jts.bundle.version>1.12_1</jts.bundle.version>
+
+        <!-- If you -->
         <karaf.version>4.1.1</karaf.version>
+        <felix.configadmin.version>1.8.8</felix.configadmin.version>
+        <!--TODO: Add our felix config admin version here-->
```

Comment:
❗️ Let's make sure we move that property up into this pom. And remove the orphaned "If you" comment above.

Reply:
✏️  Remember to add all the comments we talked about to make sure all the felix version and code we copied line up when karaf is upgraded.

Reply:
Done and done 👍